File: songshizz_bot/Songwhip/Models/SongwhipInfo.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Songwhip.Models;

public class SongwhipInfo
{
    [JsonPropertyName("type")]
    public string Type { get; set; }

    [JsonPropertyName("id")]
    public int Id { get; set; }

    [JsonPropertyName("path")]
    public string Path { get; set; }

    [JsonPropertyName("name")]
    public string Name { get; set; }

    [JsonPropertyName("url")]
    public string Url { get; set; }

    [JsonPropertyName("sourceUrl")]
    public string SourceUrl { get; set; }

    [JsonPropertyName("sourceCountry")]
    public string SourceCountry { get; set; }

    [JsonPropertyName("image")]
    public string Image { get; set; }

    [JsonPropertyName("links")]
    public HasLinks Links { get; set; }

    [JsonPropertyName("linksCountries")]
    public List<string> LinksCountries { get; set; }

    [JsonPropertyName("artists")]
    public List<Artist> Artists { get; set; }

    [JsonPropertyName("overrides")]
    public object Overrides { get; set; }
}

File: modules/songwhip_api/models/songwhip_info.go
package models

type SongwhipInfo struct {
	Type           string      `json:"type"`
	ID             int         `json:"id"`
	Path           string      `json:"path"`
	Name           string      `json:"name"`
	URL            string      `json:"url"`
	SourceURL      string      `json:"sourceUrl"`
	SourceCountry  string      `json:"sourceCountry"`
	Image          string      `json:"image"`
	Links          HasLinks    `json:"links"`
	LinksCountries []string    `json:"linksCountries"`
	Artists        []Artist    `json:"artists"`
	Overrides      interface{} `json:"overrides"`
}
